playground/terraform/modules/memorystore/variables.tf

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+variable "project_id" {
+  description = "The GCP Project ID where Playground Applications will be created"
+}
+
+variable "redis_version" {
+  description = "The GCP Project ID where Playground Applications will be created"
+  default     = "REDIS_6_X" 
+}
+
+variable "beam_playground_terraform" {
```

Comment:
Should this just be a `local` instead?

Reply:
It seems like the wrong name for this variable - renamed to terraform_state_bucket_name and added description